doc/api/stream.md

```diff
@@ -1532,6 +1532,43 @@ readable.on('data', (chunk) => {
 });
 ```
 
+### stream.Transform.by(asyncGeneratorFunction, [options])
+
+* `asyncGeneratorFunction` {AsyncGeneratorFunction} A mapping function which
+accepts a `source` iterable which can be used to read incoming data, while
+transformed data is pushed to the stream with the `yield` keyword.
+* `options` {Object} Options provided to `new stream.Transform([options])`.
+By default, `Tranfrom.by()` will set `options.objectMode` to `true`,
```

Comment:
If I understand correctly, this is the exact _opposite_ of how Transform streams usually set the `objectMode`? That seems like a foot gun to me. Also, spelling: `Tranfrom.by ---> Transform.by`

Reply:
`Transform.by` is analogous to `Readme.from` - the decision for objectMode: true was already made ( see https://nodejs.org/api/stream.html#stream_readable_from_iterable_options ).

I believe the reasoning is, iterators are not constrained to any type (e.g. buffers) and may yield anything (especially, strings), so explicitly opting out of object mode shows an awareness that only buffers may be yielded in that case. - Thanks for the typo!